### ISSUE TITLE
Security vulnerability via outdated log4js dependencies

### ISSUE THREAD

**Issue by bjankord** [nicojs/node-link-parent-bin#8]
I recently ran `npm audit` on one of my projects and saw the following report:

```
=== npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ link-parent-bin [dev]                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ link-parent-bin > log4js > streamroller > debug              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/534                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

It looks like this vulnerability has been resolved in [log4js@3.0.0](https://github.com/log4js-node/log4js-node/issues/614)

**Comment by bjankord** [comment]
Related, it looks like [@types/log4js](https://www.npmjs.com/package/@types/log4js) has been deprecated and log4js now provides its [own type definitions](https://github.com/log4js-node/log4js-node/tree/master/types).

**Comment by nicojs** [comment]
Thanks for reporting 👍 Should be fixed with `v0.2.3`
